src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacy.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import ProjectDataPrivacyContent from './projectDataPrivacyContent';
+import {DataPrivacy} from './types';
+
+const ProjectDataPrivacy = ({params, organization}: DataPrivacy) => (
+  <ProjectDataPrivacyContent params={params} organization={organization} />
+);
```

Comment:
Does this component need to apply `<Feature>` to keep people who directly navigate to the view out?

Reply:
done 💪